
Whole exome sequencing of more than 2,500 simplex families, each having a 
child with an autistic spectrum disorder reported in
<a href="https://www.nature.com/articles/nature13908" target="_blank">
Iossifov, I. et al., *Nature*, 2014.
</a>.

**Title**:The contribution of de novo coding mutations to autism spectrum disorder

**Abstract**: Whole exome sequencing has proven to be a powerful tool for understanding the
genetic architecture of human disease. Here we apply it to more than 2,500
simplex families, each having a child with an autistic spectrum disorder. By
comparing affected to unaffected siblings, we show that 13% of de novo missense
mutations and 43% of de novo likely gene-disrupting (LGD) mutations contribute
to 12% and 9% of diagnoses, respectively. Including copy number variants,
coding de novo mutations contribute to about 30% of all simplex and 45% of
female diagnoses. Almost all LGD mutations occur opposite wild-type alleles.
LGD targets in affected females significantly overlap the targets in males of
lower intelligence quotient (IQ), but neither overlaps significantly with
targets in males of higher IQ. We estimate that LGD mutation in about 400 genes
can contribute to the joint class of affected females and males of lower IQ,
with an overlapping and similar number of genes vulnerable to contributory
missense mutation. LGD targets in the joint class overlap with published
targets for intellectual disability and schizophrenia, and are enriched for
chromatin modifiers, FMRP-associated genes and embryonically expressed genes.
Most of the significance for the latter comes from affected females.

**Pubmed**: 
<a href="https://pubmed.ncbi.nlm.nih.gov/25363768/" target="_blank">25363768</a>

**Data**: <a href="https://www.nature.com/articles/nature13908#Sec31" target="_blank">
https://www.nature.com/articles/nature13908#Sec31</a>
The data from this paper comes from Supplementary Table 2.