
A set of exonic single-nucleotide variants (SNVs) from 2,377 families with 
autism reported in 
<a href="https://www.nature.com/articles/ng.3303" target="_blank">
Krumm, N. et al., *Nat Genet*, 2015
</a>.

**Title**: Excess of rare, inherited truncating mutations in autism

**Abstract**: To assess the relative impact of inherited and de novo variants on autism risk,
we generated a comprehensive set of exonic single-nucleotide variants (SNVs)
and copy number variants (CNVs) from 2,377 families with autism. We find that
private, inherited truncating SNVs in conserved genes are enriched in probands
(odds ratio = 1.14, P = 0.0002) in comparison to unaffected siblings, an effect
involving significant maternal transmission bias to sons. We also observe a
bias for inherited CNVs, specifically for small (<100 kb), maternally inherited
events (P = 0.01) that are enriched in CHD8 target genes (P = 7.4 Ã— 10(-3)).
Using a logistic regression model, we show that private truncating SNVs and
rare, inherited CNVs are statistically independent risk factors for autism,
with odds ratios of 1.11 (P = 0.0002) and 1.23 (P = 0.01), respectively. This
analysis identifies a second class of candidate genes (for example, RIMS1, CUL7
and LZTR1) where transmitted mutations may create a sensitized background but
are unlikely to be completely penetrant.

**Pubmed**:
<a href="https://pubmed.ncbi.nlm.nih.gov/25961944/" target="_blank">25961944</a>

**Data**:
<a href="https://nda.nih.gov/study.html?id=353" target="_blank">https://nda.nih.gov/study.html?id=353</a>
<a href="https://www.nature.com/articles/ng.3303#Sec23" target="_blank">https://www.nature.com/articles/ng.3303#Sec23</a>
